content/tracing/advanced_usage/_index.md

```diff
@@ -1018,6 +1055,47 @@ Coming Soon. Reach out to [the Datadog support team][contact support] to be part
 
 [contact support]: https://docs.datadoghq.com/help
 
+{{% /tab %}}
+{{% tab "C++" %}}
+
+Distributed tracing can be accomplished by [using the Inject and Extract methods on the tracer](https://github.com/opentracing/opentracing-cpp/#inject-span-context-into-a-textmapwriter), which accept [generic `Reader` and `Writer` types](https://github.com/opentracing/opentracing-cpp/blob/master/include/opentracing/propagation.h). Priority sampling (enabled by default) should be on to ensure uniform delivery of spans.
```

Comment:
```suggestion
Distributed tracing can be accomplished by [using the `inject` and `extract` methods on the tracer](https://github.com/opentracing/opentracing-cpp/#inject-span-context-into-a-textmapwriter), which accept [generic `reader` and `writer` types](https://github.com/opentracing/opentracing-cpp/blob/master/include/opentracing/propagation.h). Priority sampling (enabled by default) should be on to ensure uniform delivery of spans.
```

Reply:
The methods are capitalised. So `Inject` and `Extract` are correct. Same thing for Reader and Writer.